graphql/concrete_notification/fields.ts

```diff
@@ -39,26 +39,12 @@ export class ExtendedFieldsConcreteNotificationResolver {
         return await prisma.notification.findUnique({ where: { id: concreteNotification.notificationID } });
     }
 
-    @FieldResolver((returns) => String)
+    @FieldResolver((returns) => NotificationMessage)
```

Comment:
Have you tried this? I don't think this will work without the @Field() and @ObjectType() annotations.

Reply:
it works, but I have to query the fields